### ISSUE TITLE
Shibboleth 5.X upgrade on dev

### ISSUE THREAD

**Issue by mar235av** [UWIT-IAM/uw-idp-custom#50]
Dependent on multiple pre-upgrade issues.

Once the repository is ready, upgrade to the latest version of Shibboleth 5.X following the documented release process:

- Copy the deployment directory on dev.
- Apply the upgrade to the latest 5.X version.
- Update any plugins as needed.
- Install our custom modules re-compiled for 5.X.
- Apply required configuration updates (see below).
- Manually validate and resolve additional configuration issues as needed.

We know of two required configuration updates:
1. Update a class reference to RelyingPartyContext in conf/authn/mfa-authn-config.xml (5.0.0 release notes)
2. Remove the system/ directory (5.0.0 release notes)
There are other potential issues that would be discovered only after updating the dev environment.

**Comment by mar235av** [comment]
Completed. I performed the update steps as documented.
A number of new files (about 70) were created as part of the update. I will create a new issue to review those files.
Additional iitems that we discover will also be tracked as separate issues.

**Comment by chmc3** [comment]
This effort was completed with the release of idP v5 on 2024-08-13
